src/Components/Address/__tests__/useAddressAutocomplete.jest.ts

```diff
@@ -196,6 +199,16 @@ describe("useAddressAutocomplete", () => {
         expect(result.current.autocompleteOptions).toEqual([])
         expect(mockFetch).not.toHaveBeenCalled()
       })
+
+      it("throttles calls to the Smarty API", async () => {
+        const { result } = setupHook({ country: "US" })
+
+        act(() => {
+          result.current.fetchForAutocomplete({ search: "401 Broadway" })
+        })
+
+        expect(throttle).toHaveBeenCalled()
+      })
```

Comment:
In order to really test this we need to ask whether it was called with the private `fetchSuggestions` function defined inside the hook and used by `fetchForAutocomplete` and `fetchSecondarySuggestions`, ideally also that it was only called once (we aren't creating multiple throttles which would make it not work). I was the one who recommended this test, but in short I don't know if it's really possible to test if we are using `_.throttle`.

Reply:
We could assert something like 
```ts
expect(throttle.mock.calls.length).toEqual(1)
expect(throttle.mock.calls[0]).toEqual([expect.anything(), 500, {leading: true, trailing: true}])
```

But I think deleting this could also be fine.

Reply:
I'll include these assertions and see how things go but will delete as a backup if I can't get things to pass. Overall, I think its okay to keep this test because we aren't testing lodash but just that its being called.

Reply:
Based on how the test is written I think it should be ` expect(throttle).toHaveBeenCalledWith([
    expect.any(Function), // <-- Change here
    500,
    { leading: true, trailing: true },
  ])`

because it actually returns an anonymous function instead of anything. Also I believe Jest's `expect.anything()` matcher doesn't match against a function type but only against other non-`null` and non-`undefined` values. Jest has other matchers like `expect.any(Function)` to match against expecting a function as the first arg.